charts/cloudtty/values.yaml

```diff
@@ -92,3 +92,17 @@ cloudshellImage:
   registry: ghcr.io
   repository: cloudtty/cloudshell
   tag: "v0.8.2"
+
+cloudshell:
```

Comment:
Can the cloudshellImage be merged into the cloudshell structure? how do you think?

Reply:
I agree with you. I have fix it

Reply:
/auto-cc calvin0327